<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportWidget</class>
 <widget class="QWidget" name="ExportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>120</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>291</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>291</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDateTimeEdit" name="fromDTE">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="fromL">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>From</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="toDTE">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="toL">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QComboBox" name="formatCB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>CSV</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>JSON</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="formatL">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Format</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportPB">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
